doc/data_protocol.md

```diff
@@ -86,33 +86,90 @@ The length of the data field is variable
+> **Note:** the CAN bus between the Engine Computer and the Flight Controller can be extended to additional boards in future developments.
 
-CAN bus: CAN2.0 (FlexCAN) @1Mbit/s with termination 120Ω resistors on the Flight Controller side and on the Engine Computer side
+CAN bus: CAN2.0 (FlexCAN) @1Mbit/s with termination 120Ω resistors on the Flight Controller side and on the Engine Computer side.
 
 Base frame format (11 identifier bits)
 
+>**Note:** The Time Sync of the Engine Computer will be handled by the Flight Controller. The EC will send a request package and the FC will answer it using its currently configured system time (this can be the time set by the Ground Station or the time received through GNSS). 
+
 # Data protocol
 
-The term `ID` is used below to denote `identifier field` of the CAN Bus and byte 2 (`Frame ID`) of a frame over the serial links
+The term `ID` is used below to denote `identifier field` of the CAN Bus and byte 2 (`Frame ID`) of a frame over the serial links.
 
-The `identifier field` is an 11 bits field but only the 8 lower bits are used to make the data on the CAN Bus as similar as possible to the data on the serial buses. The remaining `ID` values `100` - `7FF` are available for data transfer between on-board systems over the CAN Bus only and cannot be sent to the ground.
+The `identifier field` is an 11 bits field but only the 8 lower bits are used to make the data on the CAN Bus as similar as possible to the data on the serial buses. The remaining `ID` values `100` - `7FF` are available for data transfer between on-board systems over the CAN Bus only, and cannot be sent to ground.
 
-Each message on the Telecommand link and the Telemetry link has a unique ID
+Each message on the Telecommand TC link and the Telemetry TM link has a unique ID.
 
-| ID              | Frame type | From                        | To                          |
-| --------------- | ---------- | --------------------------- | --------------------------- |
-| `0x00` - `0x0F` | TC         | Ground or Flight Controller | Engine Computer             |
-| `0x10` - `0x7F` | TM         | Engine Computer             | Ground or Flight Controller |
-| `0x80` - `0x8F` | TC         | Ground                      | Flight Controller           |
-| `0x90` - `0xEF` | TM         | Flight Controller           | Ground                      |
-| `0xF0` - `0xFF` | N/A        | N/A                         | N/A                         |
+<table>
+<thead>
+  <tr>
+    <th>ID</th>
+    <th>Frame Type</th>
+    <th>From</th>
+    <th>To</th>
+    <th>Unique IDs</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>0x00 - 0x0F</td>
+    <td>TC</td>
+    <td>Ground Station</td>
+    <td>Flight Controller</td>
+    <td>16</td>
+  </tr>
+  <tr>
+    <td>0x10 - 0x1F</td>
+    <td>TC</td>
+    <td>Flight Controller</td>
+    <td>Ground Station</td>
+    <td>16</td>
+  </tr>
+  <tr>
+    <td>0x20 - 0x2F</td>
+    <td>TC</td>
+    <td>Ground Station</td>
```

Comment:
```suggestion
    <td>Ground Station or Flight Controller</td>
```

Reply:
Should not the commands that go only between the FC and the EC use the 0x100 - 0x7FF address space instead? This way we can separate the on-board exclusive communications from the data relayed to ground. See the Time Sync protocol between the FC and EC.

Reply:
Messages on the CAN bus are prioritized according to their ID (lowest goes first), so anything critical should probably have lower ID than telemetry messages! 😅

Reply:
Maybe we should use different IDs for CAN messages and telemmetry links then?

Reply:
As long as we keep the mapping between IDs simple, I don't see a problem with it 👍 

Although I do think it would be cool if we could have a CAN bus on the Ground Controller as well to be able to use the testing hardware for Edda to actually control the 😄

Reply:
I'll look into it